test/clearingHouse/ClearingHouse.closePosition.test.ts

```diff
@@ -104,21 +108,28 @@ describe("ClearingHouse closePosition", () => {
             })
 
             // taker pays 0.06151334175725025 / 0.99 = 0.06213468864 quote to pay back 0.0004084104205 base
-            await clearingHouse.connect(bob).closePosition({
+            // using original sqrtPriceX96 to avoid over price limit
```

Comment:
Could we use `await addOrder(fixture, alice, 0.001, 1, upperTick, maxTick)` to prevent the `closePosition` makes the tick to maxTick? Otherwise, this is not testing closePosition.

Reply:
change to use closePosition with `sqrtPriceLimitX96`